student.rb

```diff
@@ -15,4 +15,8 @@ def classroom=(classroom)
     @classroom = classroom
     classroom.students.push(self) unless classroom.students.include?(self)
   end
+
+  def student_to_json
+    { name: @name, age: @age, id: @id }.to_json
+  end
```

Comment:
- [ ] Kindly add `parent_permission` also.

- [ ] Also in the JSON files it is adding an  `\`. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/174922027-47edaa49-02d6-4b8c-942a-a074a4604c27.png)

Example
![image](https://user-images.githubusercontent.com/38635158/174922085-9652b473-f8ea-46d8-972b-1dec71f6116f.png)

Reply:
Hi @ahmetbozaci, thanks for your review.
We already applied the changes you requested.